BUILD

```diff
@@ -77,7 +77,31 @@ linux_packages(
     depends = [
         "openjdk-8-jre"
     ],
-    modes = {
+    permissions = {
+        "./var/log/grakn/": "0777",
```

Comment:
let's be consistent with the relative path by removing the leading `./`: `var/log/grakn/"`

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/4777/commits/1456af49e17ee353ec93c7b05a844d2ff421ab7c